From: SeongJae Park <sj@kernel.org>
Date: Thu, 16 Oct 2025 13:46:12 -0700
Subject: [PATCH] Revert "Docs/admin-guide/mm/damon/usage: document
 obsolete_target file"

This reverts commit 5fc72e53ee68be3909db04f3e86ae1f670ac232c.

Signed-off-by: SeongJae Park <sj@kernel.org>
---
 Documentation/admin-guide/mm/damon/usage.rst | 13 +++----------
 1 file changed, 3 insertions(+), 10 deletions(-)

diff --git a/Documentation/admin-guide/mm/damon/usage.rst b/Documentation/admin-guide/mm/damon/usage.rst
index d8a3d6e740b3..98958975604d 100644
--- a/Documentation/admin-guide/mm/damon/usage.rst
+++ b/Documentation/admin-guide/mm/damon/usage.rst
@@ -67,7 +67,7 @@ comma (",").
     │ │ │ │ │ │ │ intervals_goal/access_bp,aggrs,min_sample_us,max_sample_us
     │ │ │ │ │ │ nr_regions/min,max
     │ │ │ │ │ :ref:`targets <sysfs_targets>`/nr_targets
-    │ │ │ │ │ │ :ref:`0 <sysfs_target>`/pid_target,obsolete_target
+    │ │ │ │ │ │ :ref:`0 <sysfs_target>`/pid_target
     │ │ │ │ │ │ │ :ref:`regions <sysfs_regions>`/nr_regions
     │ │ │ │ │ │ │ │ :ref:`0 <sysfs_region>`/start,end
     │ │ │ │ │ │ │ │ ...
@@ -264,20 +264,13 @@ to ``N-1``.  Each directory represents each monitoring target.
 targets/<N>/
 ------------
 
-In each target directory, two files (``pid_target`` and ``obsolete_target``)
-and one directory (``regions``) exist.
+In each target directory, one file (``pid_target``) and one directory
+(``regions``) exist.
 
 If you wrote ``vaddr`` to the ``contexts/<N>/operations``, each target should
 be a process.  You can specify the process to DAMON by writing the pid of the
 process to the ``pid_target`` file.
 
-Users can selectively remove targets in the middle of the targets array by
-writing non-zero value to ``obsolete_target`` file and committing it (writing
-``commit`` to ``state`` file).  DAMON will remove the matching targets from its
-internal targets array.  Users are responsible to construct target directories
-again, so that those correctly represent the changed internal targets array.
-
-
 .. _sysfs_regions:
 
 targets/<N>/regions
-- 
2.47.3

